components/Hero.tsx

```diff
@@ -29,13 +29,13 @@ export default function Hero() {
         </p>
         <div className="grid grid-cols-1 justify-center pt-5 pb-2 sm:pb-8 md:grid-cols-2 md:space-x-5 md:pt-8">
           <div className="flex items-center justify-center md:justify-end">
-            <Button classes="text-xl w-full" href={routes.chaptersUrl}>
+            <Button classes="text-xl !w-64" href={routes.chaptersUrl}>
               {t('hero.start_journey')}
             </Button>
           </div>
           <div className="flex justify-center pt-5 md:justify-start md:pt-0">
             <Button
-              classes="text-xl w-full"
+              classes="text-xl !w-64"
```

Comment:
same here
``` suggestion
              classes="text-xl w-full md:w-64"
```

Reply:
upon removing the `!` important tag the w-auto(false) or w-full(true) class from https://github.com/saving-satoshi/saving-satoshi/blob/master/shared/Button.tsx#L30 the button component gets added. If I provide a value to the `full` prop it either gives the option for auto or full width and in case of not providing a value to the `full` prop it defaults to `w-auto` due to this ternary operation: 
```
full ? 'w-full' : 'md:w-auto'
```
In the local environment the width works without adding the important tag but in production, it disables the `w-64`class, and the `w-auto` gets enabled, hence the need for an important tag. I have added the w-full for mobile view.